src/components/Login/Register.tsx

```diff
@@ -117,16 +123,20 @@ const Register: React.FC<Props> = (props: Props) => {
                 />
               </FormControl>
             </Flex>
-            <PwdInputGroup
-              label="Password"
-              value={password}
-              onChange={(event) => setPassword(event?.currentTarget.value)}
-            />
-            <PwdInputGroup
-              label="Confirm Password"
-              value={confirmPassword}
-              onChange={(event) => setConfirmPassword(event?.currentTarget.value)}
-            />
+            <div style={{ marginTop: "8px" }}>
```

Comment:
_marginTop_ can be added in _PwdInputGroup/index.tsx_, so you don't have to change it everywhere
![image](https://user-images.githubusercontent.com/76163407/199658770-8825dc5a-be88-4422-be0f-af72dd378faf.png)

Reply:
Understood! Thanks Shawn!